lib/rest.js

```diff
@@ -99,7 +89,7 @@
 			rejectUnauthorized: process.env.PIPEDRIVE_API_HOST == 'api.pipedrive.com' ? true : false
 		};
 
-		log('Performing API request: ' + JSON.stringify(_.extend(requestOptions, { url: url }), null, 4));
+		log('Adding API request to queue: ' + JSON.stringify(_.extend(_.omit(requestOptions, this.options.multipart ? ['payload'] : []), { url: url }), null, 4));
```

Comment:
What queue?

Reply:
@tot-ra The request is pushed into `requestQueue` two lines down. So if rate limiting kicks in the request is actually not immediately made.